listener/Dockerfile

```diff
@@ -223,9 +223,10 @@ COPY --from=build_image /project/avalon/listener/dist/*.whl dist/
 # So remove python3-pip package and dependencies after installing wheel file.
 RUN apt-get update \
  && apt-get install -y -q python3-pip \
- && pip3 install json-rpc pyzmq\
+ && pip3 install json-rpc pyzmq \
  && echo "Install Common Python, Crypto, Listener and SDK packages\n" \
  && pip3 install dist/*.whl \
+ && pip3 install pycryptodomex ecdsa \
```

Comment:
Add `pip3 install pyzmq pycryptodomex ecdsa` to the BUILD.md instructions under "Install PIP3 packages"